js/components/data-sources.js

```diff
@@ -1093,11 +1094,11 @@ define(['jquery', 'knockout', 'atlas-state', 'text!./data-sources.html', 'd3', '
 			var minValue = histogramData.min;
 			var intervalSize = histogramData.intervalSize;
 
-			for (var i = 0; i <= histogramData.intervals; i++) {
+			for (var i = 0; i < histogramData.data.intervalIndex.length; i++) {
 				var target = new Object();
-				target.x = minValue + 1.0 * i * intervalSize;
+				target.x = minValue + (1.0 * histogramData.data.intervalIndex[i] * intervalSize);
```

Comment:
@chrisknoll, intervalIndex contains list of ages, e.g. interval from 18 to 90, so it doesn't make sense to add `minValue` to the indexes

Reply:
In that case, the minValue will be 0.  The wider usecase of supplying a minValue is where the histogram data is comming in as 0-indexed, but you want the X-axis of the report to start at some non-zero x-coordinate value.

For example: if you recieved histogram data for 'year 1 after studyStart'...'year 2 after studyStart'..etc, but you know the study started in 2005 you would set the minValue to 2005, and use the histogram index as the 2005+index to give you the bar's position on the hisogram.  

In the cases of our age histograms, we know the indexes don't need to be offset, so the minValue is 0 so not to offset anything.

Note, this code is done in 'map histogram' so no assumptions can be made there about what the nature of the indexes are.  The minValue is just an option that lets you specify that the index value should map to another actual value on the X coordinates.  You can see in the code for 'age at first' we're setting the min/max based on the values of the indexes (because thsoe indexes have actual meanings about the ages.